scipy/stats/_probability_distribution.py

```diff
@@ -123,11 +123,19 @@ def sample(self, shape, *, method, rng):
             Not all `method` options are available for all distributions.
             If the selected `method` is not available, a `NotImplementedError``
             will be raised.
-        rng : `numpy.random.Generator`, optional
-            Pseudorandom number generator state. When `rng` is None, a new
-            `numpy.random.Generator` is created using entropy from the
-            operating system. Types other than `numpy.random.Generator` are
-            passed to `numpy.random.default_rng` to instantiate a `Generator`.
+        rng : `numpy.random.Generator` or `scipy.stats.QMCEngine`, optional
+            Pseudo- or quasi-random number generator state. When `rng` is None,
+            a new `numpy.random.Generator` is created using entropy from the
```

Comment:
Just to entertain the thought for 1 second. Did you consider making Halton the default vs MC? There is a slight tradeoff between speed and quality as the shape goes crazy I suppose.

Reply:
We discussed offline that in some cases, such as computational hypothesis testing, you want samples to have independent observations, not low-discrepancy sequences. While QMC is definitely better in some circumstances, it comes with gotchas, so the user should opt in.